README.md

```diff
@@ -2,8 +2,78 @@
 
 [![Build Status](https://travis-ci.org/sourcecred/sourcecred.svg?branch=master)](https://travis-ci.org/sourcecred/sourcecred)
 
-The open-source community provides an enormous amount of value to the world. However, open-source contributors go largely unrewarded and unrecognized. SourceCred aims to help that situation, by building tools that enable quantitatively measuring the value that open-source contributors provide to individual projects, and to the community as a whole.
+## Vision
 
-SourceCred will create a "Cred Graph", which is a graph that shows how the contributions that compose open-source projects are related to and derive value from each other. From this, we'll be able to assign "cred" to users based on how valuable their contributions are. Cred will be assigned based on a mixture of objective data (e.g. references between GitHub pull requests) and subjective feedback (e.g. projects' own judgments on how important different contributions were).
+Open source software is amazing, and so are the creators and contributors who
+share it. How amazing? It's difficult to tell, since we don't have good tools
+for recognizing those people. Many amazing open-source contributors labor in
+the shadows, going unappreciate for the work they do.
 
-If you'd like to contribute, please follow along with [our issues](https://github.com/sourcecred/sourcecred/issues), as we are using issues to coordinate development and design decisions. We also have [a slack](https://join.slack.com/t/sourcecred/shared_invite/enQtMzA4NzI5ODIwODMyLWFiNDlhNWNiODc4MTk4MjNmZTAzMDNjNDAwYzEyZTBiNjAxZTFhMjU1MDg2YzNlN2FlNzgwYmU0NGM1NGEzM2M).
+As the open economy develops, we need to go beyond [commit streaks] and
+follower counts. We need transparent, accurate, and fair tools for recognizing
+and rewarding open collaboration. SourceCred aims to do that.
+
+[commit streaks]: https://www.mxsasha.eu/blog/2016/04/01/how-github-contribution-graph-is-harmful/
+
+SourceCred will enable projects to create and track "cred", which is a
+quantitative measure of how much value different contributors added to a
+project. We'll do this by providing a basic data structure—a [cred graph]— into
```

Comment:
nit: em dash should not be followed by space

Reply:
Done